app/components/modal/CancelRequest.tsx

```diff
@@ -48,11 +58,32 @@ export const openRequestCancelModal = createModal<ModalProps, ModalState>({
         <p className="text-sm text-gray-900 mb-4">
           Are you sure you want to cancel this request? This action cannot be undone!
         </p>
+        {isPublicAdmin && (
```

Comment:
to make things simpler, let's make the `decisionComment` mandatory when cacelling requests whoever does it